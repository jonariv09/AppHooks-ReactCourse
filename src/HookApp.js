import React from 'react';

function HookApp() {
  return (
    <div className="App">
      <h1> Hola mundo </h1>
    </div>
  );
}

export default HookApp;
