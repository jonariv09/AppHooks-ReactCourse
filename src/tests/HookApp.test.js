import '@testing-library/jest-dom';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import HookApp from '../HookApp';

describe('HookApp', () => {
  test('debe renderizar correctamente el componente', () => {
    const wrapper = shallow(<HookApp />);
    expect(toJson(wrapper)).toMatchSnapshot();
  })
}) 
