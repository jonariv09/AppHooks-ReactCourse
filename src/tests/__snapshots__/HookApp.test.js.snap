// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HookApp debe renderizar correctamente el componente 1`] = `
<div
  className="App"
>
  <h1>
     Hola mundo 
  </h1>
</div>
`;
